packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -13,166 +13,189 @@ import {
     fillSelectedColor,
     iconColor,
     smallDelay,
-    standardPadding
+    standardPadding,
+    buttonPrimaryFontColor,
+    buttonFillPrimaryColor,
+    buttonFillPrimaryActiveColor,
+    buttonFillAccentColor,
+    buttonAccentBlockFontColor,
+    buttonFillAccentActiveColor,
+    buttonBorderAccentOutlineColor,
+    buttonAccentOutlineFontColor
 } from '../../theme-provider/design-tokens';
 import { appearanceBehavior } from '../../utilities/style/appearance';
 import { ButtonAppearance } from './types';
 
 export const styles = css`
+    @layer default, hover, focusVisible, active, disabled;
+
     ${display('inline-flex')}
```

Comment:
move this into the default layer (it just emits a full style on it's own)

Reply:
re-opening because there was a mixup. The display style needs to move into the default layer, we can keep the layer definition outside

```css
@layer base, hover, focusVisible, active, disabled, top;

@layer base {
    ${display('inline-flex')}

    :host {
            background-color: transparent;
   ...
   }
}
```